### DISCUSSION TITLE
[Question]: How to use Home Assistant Integration Notification API

### DISCUSSION THREAD

**Discussion by e10kstarfire** [blakeblackshear/frigate#13268] [Ask A Question] [answered]
### What is your question:

I am trying to use the Home Assistant integration Notification API as described here https://docs.frigate.video/integrations/home-assistant#notification-api

I run frigate in a docker container and HA on a RPi

I cannot get the URL described in the docs linked above to work, when I click on the link generated I get a webpage that says:
message: "event not found"
success:"false"

Can't figure it out, I am using my HA URL, and obtaining the event id via {{trigger.payload_json["after"]["id"]}}

The full URL resolves for example to:

`https://<HA-URL>/api/frigate/notifications/1724324537.227766-mhzjbr/clip.mp4`

Any ideas please?

**Comment by e10kstarfire** [comment]
Ah never mind, I think I know the issue:

I am processing MQTT frigate/reviews events, and the id returned from that in the JSON looks like a review ID and not an event ID...


So not sure how to proceed as event ID is provided in frigate/reviews JSON but there can be multiple...


Not sure how I can really use this Notification API now

**Reply by NickM-27** [reply, marked as answer]
id is a review id, the detections field contains all of the event ids
